qt/python/mantidqt/widgets/plotconfigdialog/axestabwidget/__init__.py

```diff
@@ -37,10 +37,15 @@ def from_ax_object(cls, ax):
     def from_view(cls, view):
         props = dict()
         props['title'] = view.get_title()
-        props['xlim'] = (view.get_xlower_limit(), view.get_xupper_limit())
+
+        try:
```

Comment:
Could you help me understand why the `try...except` is necessary? Doesn't the `QDoubleValidator` ensure that we always have a `float`?